### ISSUE TITLE
Switching to and Operator with more Operands

### ISSUE THREAD

**Issue by sformichella** [Math-Problems-API/front-end#21]
Selecting an operator with more operands than the current operator breaks stuff.

This is because the second operand `RightOperand` is undefined which is caused by a delay in how the operands array is computed. 

First, a number `numberOfOperands` is computed from the operator state. Then, the operands array is set to an array containing this number of operands in a `useEffect` that depends on `numberOfOperands`

In short, the operands array doesn't update fast enough. 

Perhaps, a solution would be to separate the GraphQL operands from the operands in the selects. Make a higher level GraphQL state that lives in `App` and a select-level state that lives inside of operators.
